### PR TITLE
add standard Dockerfile

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/5980.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.8
```
Comment (MarcelMue) [inline Dockerfile]: I think there is 3.9 by now but no big deal.

**Review by MarcelMue — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (xh3b4sd) [reply Dockerfile]: YOLO

**Review by rossf7 — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+RUN apk add --update ca-certificates \
+    && rm -rf /var/cache/apk/*
```
Comment (TheoBrigitte) [inline Dockerfile]: You can also use `--no-cache` instead of `--update` and avoid the `rm -rf /var/cache/apk/` line. Feels cleaner.

**Review by TheoBrigitte — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (xh3b4sd) [reply Dockerfile]: Does SIG RelEng want to document this or sort some tooling to have it consistent for all projects?

Comment (rossf7) [reply Dockerfile]: I'm happy to write some docs in the SIG section of the giantswarm repo.

@xh3b4sd @MarcelMue @TheoBrigitte WDYT?

Comment (MarcelMue) [reply Dockerfile]: I created a card on the releng board. If you want to initiate writing docs on this topic, then I am more then happy to review / help out.

Comment (rossf7) [reply Dockerfile]: @MarcelMue Ack. Thx!

**Pull request merged**